### PR TITLE
ref: classes refactor

### PR DESCRIPTION
**Pull request by Dav1s-Ops**
Refactor takes care of CSV generator methods and eliminates unneeded attributes.

### CODE REVIEW

**Review by kcockett — APPROVED** [review]
As we collaborated together on this branch, all changes look good and tested.

File: lib/game.rb [hunk 1]
```diff
@@ -1,32 +1,18 @@
 class Game
   attr_reader :game_id, 
               :season, 
-              :type,
```
Comment (WZale) [inline lib/game.rb:4]: YAGNI! Excellent.

**Review by WZale — APPROVED** [review]
This is definitely cleaner. I like it!

**Review by Gabe-Torres — APPROVED** [review]
Nice! Cleaned it up and it looks good

**Pull request merged**